draft-ietf-tls-esni.md

```diff
@@ -690,6 +681,10 @@ If the server sends an "encrypted_client_hello" extension, the client MUST check
 the extension syntactically and abort the connection with a "decode_error" alert
 if it is invalid.
 
+[[OPEN ISSUE: if the client sends a GREASE "encrypted_client_hello" extension, 
+should it also send a GREASE "pre_shared_key" extension? If not, GREASE+ticket
+is a trivial distinguisher.]]
```

Comment:
I suggest leaving this OPEN ISSUE open and address it post -08. This is one way that real ECH sticks out from covertext, but there are likely others. What is needed is a big picture "don't stick out" review of ECH.

Reply:
I'm fine with that, I think, though we may want to reconsider it before the GREASE experiments commence.